cocktails.mcp/src/cmd/main.go

```
@@ -19,8 +26,38 @@ func main() {
+			rec := &statusRecorder{ResponseWriter: w, status: 200}
+			next.ServeHTTP(rec, r)
+			log.Printf("%s %s %s %d", r.Method, r.URL.Path, r.RemoteAddr, rec.status)
+		})
+	}
+
+	if *httpAddr != "" {
+		// HTTP mode
+		http.HandleFunc("/healthz", func(w http.ResponseWriter, r *http.Request) {
+			w.Header().Set("Content-Type", "application/json")
+			w.WriteHeader(http.StatusOK)
+			w.Write([]byte(`{"status":"ok"}`))
```

golangci-lint (1.64.8)

Error return value of `w.Write` is not checked

(errcheck)